generator/lib/data.ml

```diff
@@ -1,14 +1,13 @@
 open Idl
 open Common
 
+(** A string representing a Xen domain on the local host. The string is
+    guaranteed to be unique per-domain but it is not guaranteed to take
+    any particular form. It may (for example) be a Xen domain id, a Xen
+    VM uuid or a Xenstore path or anything else chosen by the toolstack.
+    Implementations should not assume the string has any meaning. *)
 type domain = string
 [@@deriving rpcty]
-[@@doc [
-  "A string representing a Xen domain on the local host. The string is";
-  "guaranteed to be unique per-domain but it is not guaranteed to take";
-  "any particular form. It may (for example) be a Xen domain id, a Xen";
-  "VM uuid or a Xenstore path or anything else chosen by the toolstack.";
-  "Implementations should not assume the string has any meaning."]]
```

Comment:
i was under the impression that these tags generate docs for the IDL? but i could be completely wrong